### ISSUE TITLE
Update `runCmd` to parse the commands defined in the working document

### ISSUE THREAD

**Issue by tvanesse** [tvanesse/PVCharacterization#1]
The current implementation of [`runCmd`](https://github.com/tvanesse/PVCharacterization/blob/master/firmware/src/pvclib.cpp#L104) does not take into account the set of low-level functions that were defined in the working document.

This function should be updated in order to integrate our specifications by:
- defining a command syntax such as <CMD>[ARGS,] (e.g., "G13.494,67.399" to make a `goTo(13.494, 67.399);`)
- implement the parsing for each valid function

**Comment by tvanesse** [comment]
Ping @ahmed-akl and @snowy13 . If it is OK for you we can use the Issue system to track the small tasks that need to be addressed and make sure we don't do the job twice :wink:  Do you agree?

**Comment by ahmed-akl** [comment]
i agree on any tracking system that prevents double work, but what is the
Issue system?

**Comment by tvanesse** [comment]
You are actually writing in it :-) The issue system is a service provided by Github to easily track the bugs and tasks related to a repo.
